src/main/resources/com/openshift/jenkins/plugins/pipeline/dsl/OpenShiftImageStreams/help-apiURL.html

```diff
@@ -0,0 +1,6 @@
+<div>
```

Comment:
can't tell from the review tool - do you create new files for all the different help files or link this file to the analogous file in the classic job package ...i've been able to link help files to avoid duplication